### PR TITLE
remove duplicate bronchioalveolar stem cell

### PR DESCRIPTION
**Pull request by shawntanzk**
Fixes https://github.com/obophenotype/cell-ontology/issues/1443

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -25945,12 +25946,14 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0011679"^^xsd:string) obo:
...
 EquivalentClasses(obo:CL_1001561 ObjectIntersectionOf(obo:CL_0000101 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002255) ObjectSomeValuesFrom(obo:RO_0013007 obo:UBERON_0003367)))
 
-# Class: obo:CL_1001566 (bronchioalveolar stem cells)
+# Class: obo:CL_1001566 (obsolete bronchioalveolar stem cells)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0011691"^^xsd:string) obo:IAO_0000115 obo:CL_1001566 "A respiratory stem cell found at the junction of the terminal (conductive) bronchiole and the respiratory bronchiole, which gives rise to alveolar cell types and club cells in response to lung injury."^^xsd:string)
-AnnotationAssertion(rdfs:label obo:CL_1001566 "bronchioalveolar stem cells"^^xsd:string)
-SubClassOf(obo:CL_1001566 obo:CL_0000034)
-SubClassOf(obo:CL_1001566 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000115))
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0011691"^^xsd:string) obo:IAO_0000115 obo:CL_1001566 "OBSOLETE A respiratory stem cell found at the junction of the terminal (conductive) bronchiole and the respiratory bronchiole, which gives rise to alveolar cell types and club cells in response to lung injury."^^xsd:string)
```
Comment (nicolevasilevsky) [inline src/ontology/cl-edit.owl:25951]:
```suggestion
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0011691"^^xsd:string) obo:IAO_0000115 obo:CL_1001566 "OBSOLETE. A respiratory stem cell found at the junction of the terminal (conductive) bronchiole and the respiratory bronchiole, which gives rise to alveolar cell types and club cells in response to lung injury."^^xsd:string)
```
This is super minor - I think we want to add a period after OBSOLETE in the definition. I updated this in the OBO Academy documentation:
https://oboacademy.github.io/obook/howto/obsolete-term/#obsoletion-process-manual

Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl:25951]: it won't let me commit this suggestion, will it let you do it @shawntanzk?

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:25951]: hmmm strange cause I used the wizard, oh well, either way I've committed the suggestion :) thanks!

Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl:25951]: Oh interesting, does the wizard add the OBSOLETE to the def? (I never use the wizard, I always use Edit -> Make entity obsolete

Comment (nicolevasilevsky) [reply src/ontology/cl-edit.owl:25951]: > guess not too, no worries, ill fix it outside

too bad! The inline commit function is so cool but it doesn't always work and I don't understand why.

**Pull request merged**